compiler/src/dotty/tools/io/AbstractFile.scala

```diff
@@ -107,18 +108,26 @@ abstract class AbstractFile extends Iterable[AbstractFile] {
   def container : AbstractFile
 
   /** Returns the underlying File if any and null otherwise. */
-  def file: JFile
+  def file: JFile = try {
```

Comment:
Ideally we should remove this

Reply:
That will be for next step. That might need changes outside of the Path/File/Directory abstractions. I would prefer to keep the changes only inside these abstractions for this first step.